.config/rollup.dist.config.mjs

```diff
@@ -380,10 +380,10 @@ export default () => {
               )
           ],
           [
-            TAP,
+            VITEST,
             () =>
-              // Lazily access constants.ENV[TAP].
-              !!constants.ENV[TAP]
+              // Lazily access constants.ENV[VITEST].
+              !!constants.ENV[VITEST]
```

Comment:
No longer need to add this as it's added before.

Reply:
So the TAP one is redundant too?

Reply:
gonna merge. if redundant we can drop it in a separate PR. it doesnt seem to block right now